### PR TITLE
Display connectivity info without obstructing current map view

### PR DESCRIPTION
**Pull request by akhuoa**
The original task was to show the provenance info in the sidebar.
[Wrike ticket](https://www.wrike.com/open.htm?id=1382421956)
There are 3 PRs related to this.

1. https://github.com/ABI-Software/flatmapvuer/pull/175
2. https://github.com/ABI-Software/mapintegratedvuer/pull/223

### CODE REVIEW

File: src/components/SideBar.vue [hunk 1]
```diff
@@ -91,7 +104,10 @@ export default {
      */
     tabs: {
       type: Array,
-      default: () => [{ title: 'flatmap', id: 1 }],
+      default: () => [
+        { title: 'Search', id: 1 },
+        { title: 'Connectivity', id: 2 }
+      ],
```
Comment (alan-wu) [inline src/components/SideBar.vue:110]: Can we add a type (with the value being search or connectivity) field which would make it clearer and will probably make the logic simpler now and in the future if we decide to have multiple search/connectivity tabs.

Comment (alan-wu) [reply src/components/SideBar.vue:110]: Looking deeper into the codes, adding a type field will help with the logic at multiple places.

**Review by alan-wu — CHANGES_REQUESTED** [review]
Looks good, please see suggestions. Having a type field to clarify the content of a tab would replace using a hardcoded id number at a number of locations.

File: src/components/SideBar.vue (hunk 1 above)
Comment (akhuoa) [reply src/components/SideBar.vue]: Tab type is added and it also supports multiple search tabs and multiple connectivity tabs.

**Comment by akhuoa** [comment]
> Looks good, please see suggestions. Having a type field to clarify the content of a tab would replace using a hardcoded id number at a number of locations.

The tabs are now with `type` fields.

**Review by alan-wu — APPROVED** [review]
Looks good to me.

**Pull request merged**